truckersmp_cli/steamcmd.py

```diff
@@ -120,7 +121,9 @@ def update_game():
         logging.debug("Closing Linux version of Steam")
         subproc.call(("steam", "-shutdown"))
     # Windows version of Steam
-    if wine and check_steam_process(use_proton=False, wine=wine, env=env_steam):
+    if (Args.check_windows_steam and
+            wine and
+            check_steam_process(use_proton=False, wine=wine, env=env_steam)):
```

Comment:
Use the same coding style as `handle_starttag()` in `truckersmp.py`:
```suggestion
    if (Args.check_windows_steam
            and wine
            and check_steam_process(use_proton=False, wine=wine, env=env_steam)):
```